### PR TITLE
ETE uses cimg/openjdk:11.0

### PR DESCRIPTION
**Pull request by schlosna**
## General
**Before this PR**:
ETE was using deprecated AdoptOpenJDK container images (see https://github.com/AdoptOpenJDK/openjdk-docker/issues/238#issuecomment-1409512318 )

**After this PR**:

==COMMIT_MSG==
ETE uses amd64/eclipse-temurin:11-jdk

Migrate off deprecated adoptopenjdk container images to Eclipse Temurin
==COMMIT_MSG==

**Priority**:

**Concerns / possible downsides (what feedback would you like?)**:

**Is documentation needed?**:

## Compatibility
**Does this PR create any API breaks (e.g. at the Java or HTTP layers) - if so, do we have compatibility?**:

**Does this PR change the persisted format of any data - if so, do we have forward and backward compatibility?**:

**The code in this PR may be part of a blue-green deploy. Can upgrades from previous versions safely coexist? (Consider restarts of blue or green nodes.)**:

**Does this PR rely on statements being true about other products at a deployment - if so, do we have correct product dependencies on these products (or other ways of verifying that these statements are true)?**:

**Does this PR need a schema migration?**

## Testing and Correctness
**What, if any, assumptions are made about the current state of the world? If they change over time, how will we find out?**:

**What was existing testing like? What have you done to improve it?**:

**If this PR contains complex concurrent or asynchronous code, is it correct? The onus is on the PR writer to demonstrate this.**:

**If this PR involves acquiring locks or other shared resources, how do we ensure that these are always released?**:

## Execution
**How would I tell this PR works in production? (Metrics, logs, etc.)**:

**Has the safety of all log arguments been decided correctly?**:

**Will this change significantly affect our spending on metrics or logs?**:

**How would I tell that this PR does not work in production? (monitors, etc.)**:

**If this PR does not work as expected, how do I fix that state? Would rollback be straightforward?**:

**If the above plan is more complex than “recall and rollback”, please tag the support PoC here (if it is the end of the week, tag both the current and next PoC)**:

## Scale
**Would this PR be expected to pose a risk at scale? Think of the shopping product at our largest stack.**:

**Would this PR be expected to perform a large number of database calls, and/or expensive database calls (e.g., row range scans, concurrent CAS)?**:

**Would this PR ever, with time and scale, become the wrong thing to do - and if so, how would we know that we need to do something differently?**:

## Development Process
**Where should we start reviewing?**:

**If this PR is in excess of 500 lines excluding versions lock-files, why does it not make sense to split it?**:

**Please tag any other people who should be aware of this PR**:
@jeremyk-91
@sverma30
@raiju

### CODE REVIEW

File: atlasdb-ete-tests/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM adoptopenjdk/openjdk11:x86_64-debian-jre-11.0.18_10
+FROM amd64/eclipse-temurin:11-jdk
```
Comment (schlosna) [inline atlasdb-ete-tests/Dockerfile:1]: we may want to just use the same as circle, e.g.

https://github.com/palantir/atlasdb/blob/aa3429ba2543c118efaac19d6eb27b57b2e20c70/.circleci/config.yml#L6

```suggestion
FROM cimg/openjdk:11.0.10-node
```

Comment (jeremyk-91) [reply atlasdb-ete-tests/Dockerfile:1]: Wasn't aware we use node in the build process (maybe for the docs?). But either of these looks fine to me

**Comment by jeremyk-91** [comment]
👍

File: atlasdb-ete-tests/Dockerfile (hunk 1 above)
Comment (schlosna) [reply atlasdb-ete-tests/Dockerfile:1]: Our circle templates include node for web builds, though I think for this we could just use cimg/openjdk

Comment (schlosna) [reply atlasdb-ete-tests/Dockerfile:1]:
```suggestion
FROM cimg/openjdk:11.0
```

File: atlasdb-ete-tests/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM adoptopenjdk/openjdk11:x86_64-debian-jre-11.0.18_10
+FROM cimg/openjdk:11.0
```
Comment (schlosna) [inline atlasdb-ete-tests/Dockerfile:2]: https://circleci.com/developer/images/image/cimg/openjdk

**Comment by jeremyk-91** [comment]
This seems like something we'd want! 👍

**Comment by barisoyoruk** [comment]
https://github.com/palantir/atlasdb/pull/6872

**Pull request closed without merging**